notes/fix-redirects.md

```
@@ -0,0 +1,83 @@
+
+Summary:
+Total files 🔍 - 50
+Files fixed ✅ - 2  
+Files skipped ⏭️  - 48
+```
+
+## Best practices
+
+1.  Before running
+
+  *   Commit current changes
```

Markdownlint

Expected: 0; Actual: 2
Unordered list indentation

(MD007, ul-indent)